containers/azure-iotedge-dotnetcore-3.1/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet/.devcontainer/base.Dockerfile
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-3.1
+
+# Install Docker CE
+COPY library-scripts/*.sh /tmp/library-scripts/
+RUN \
```

Comment:
I think we might be able to optimize the Dockerfile - have you checked out [best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)? Also more `RUN` statements [create more layers](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers).